### DISCUSSION TITLE
I need help from the entire supertux fandom to get this thought of stealing assets out of the way

### DISCUSSION THREAD

**Discussion by VincentAraneo** [SuperTux/supertux#2926] [General]
Most likely, they will delete it or take it as a joke, but I need help, I want to try to stop stealing assets so that no one else hates me. Mr.Gaming was the one who gave me a tip or question to stop stealing assets, he said that I could make my own assets for my mod and just like that stop stealing assets, is to replace the tiles of II v2 and remaster them with a new design. Here are some screenshots (and sorry that the messages are in Spanish, since they spoke to me from my Discord server)

![mrgamingtip1](https://github.com/SuperTux/supertux/assets/139823928/2e265035-1cad-4f5c-88a3-5d61da5b90f3)
![mrgamingtip2](https://github.com/SuperTux/supertux/assets/139823928/90f77379-cfb4-48df-ae12-4abf2af3c3ea)
![mrgamingtip3](https://github.com/SuperTux/supertux/assets/139823928/3c827e47-16a2-4b8a-8778-19211fada7f1)
![mrgamingtip4](https://github.com/SuperTux/supertux/assets/139823928/2f260c76-af14-47e9-a04f-4c5c39c85061)
![mrgamingtip5](https://github.com/SuperTux/supertux/assets/139823928/0eae91b2-e07d-478e-83fa-b70a299a8874)
![mrgamingtip6](https://github.com/SuperTux/supertux/assets/139823928/06569337-15c4-429f-b160-610dd0ec82c5)

If you didn't understand, you can use a translator. That's what I need from the fandom, to help me or give me advice to be able to improve, since I'm tired of continuing to do this thing of stealing anything, and I need your help to be able to improve things and stop stealing content. I seriously need your help to stop doing this, they will most likely say that this message is just for attention, but no. it's just because I need your help, and for you to give me advice like Mr.Gaming did, and if you help me, I'll help you guys with anything the fandom needs, and if you help me too I'd be proud of the entire Supertux fandom.

One thing that I didn't put in the screenshots of the MrGaming messages that he sent me, is that one of the messages I sent, I said that all of you didn't support me at the time, and I said that you were just like my classmates at school, since they did the same thing. They didn't support me like you do, but MrGaming gave me the advice not to listen to my peers who didn't support me so I wouldn't be a bitter like them.

Several of you will tell me that I am sending this message to have attention, but I am sending it to help me. If you help me, I'll help you more, if you need a music I'll do it, or if you want a background I'll do it, Even if the supertux development team has a problem or wants collaborators, I help them without problems, that's what I'm for!, to help the necessary people with any problem they have at their levels. It will help everyone to grow the fandom more, but for that, they have to help me stop stealing assets or give me advice without throwing insults at me. Just be nice to me and don't insult me to all mercy. So if you help me, I'll help you. Believe me, I wouldn't hurt the fandom at all, I just want to help. Well, that's what I wanted to send so that more people can help me. And nothing, goodbye.

**Comment by komaros1** [comment]
LMAO

**Reply by MatusGuy** [reply]
please don't get involved unless you want to add an actual comment

**Reply by komaros1** [reply]
I'm already one time people thought that I was vincent
